artifact/local/fs.go

```diff
@@ -165,3 +165,7 @@ func (a Artifact) Inspect(ctx context.Context) (types.ArtifactReference, error)
 		BlobIDs: []string{cacheKey},
 	}, nil
 }
+
+func (a Artifact) Clean(reference types.ArtifactReference) error {
+	return a.cache.DeleteBlobs([]string{reference.ID})
```

Comment:
It should be reference.BlobIDs technically. In filesystem scanning, `artifactID` and `blobIDs` are the same "by chance". They are usually different and the method called here is "DeleteBlobs". Passing `blobIDs` makes sense.
https://github.com/aquasecurity/fanal/blob/3519fe6e6c218e32a87b1912132d59891c171cfb/types/artifact.go#L94

Reply:
updated